app/generic/messaging.py

```diff
@@ -22,7 +22,7 @@ class TrustPingMsg(BaseModel):
     comment: str
 
 
-@router.post("/send-message", status_code=204)
```

Comment:
@lohanspies I found that `send-message` route had a custom response status code `204`, causing it to deviate from the acapy spec. For consistency I'm reverting it back to `200`.